src/layout/styles/_tabbing.scss

```diff
@@ -19,6 +19,27 @@ header[data-layout=tabbing] {
   flex: 0 0 auto;
   border-bottom: 1px solid $phovea_layout_active_tab_header;
 
+  &.frozen {
+    position: absolute;
+    width: 100%;
+    z-index: 10;
+    height: 3px;
+
+    > span {
+      display: none;
+    }
+
+    &:not(.hidden-header) {
+      height: auto;
+    }
+
+    &.hidden-header {
+      * { // hide all buttons inside the header unless it should be shown with the show-header class
```

Comment:
I assume you can limit the selector to the direct children using the `>` like so: `.hidden-header > * { display: none; }`